tests/coverage/coverage.c

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
need a license here, eventually this should go into ext/ depending on the license.

Reply:
if the code was change for Zephyr, you need to document your changes to the original code